02-JS-Intro-jQuery/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Javascript</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link href="normalize.css" rel="stylesheet">
+        <link href="style.css" rel="stylesheet">
+        <script src="jquery-2.1.4.min.js"></script>
```

Comment:
Try to organize your project, for instance, all js file into a js/ folder, same for css, images, or medias in general.